PrebidMobile/PrebidMobileRendering/PluginRenderer/PrebidMobilePluginRegister.swift

```diff
@@ -0,0 +1,91 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
```

Comment:
<img width="722" alt="Screenshot 2024-05-16 at 11 05 18" src="https://github.com/prebid/prebid-mobile-ios/assets/99195905/1c130c09-2247-4826-9ac1-8a652a2535f6">

Some things don't match with the class diagram:

- No function for unregistering event listener and containsPlugin
- getRTBListOfRenderers suppose to return a list of PrebidMobilePluginRenderer